internal/super/generate_test.go

```diff
@@ -40,13 +40,33 @@ func TestGenerateNewContract(t *testing.T) {
 	assert.NoError(t, err, "Failed to read generated file")
 	assert.NotNil(t, fileContent)
 
+	testContent, err := state.ReaderWriter().ReadFile("cadence/tests/TestContract_test.cdc")
+	assert.NoError(t, err, "Failed to read generated file")
+	assert.NotNil(t, testContent)
+
 	// Check content is correct
 	expectedContent := `
 access(all)
 contract TestContract {
     init() {}
 }`
+
+	expectedTestContent := `import Test
```

Comment:
is this the same as `testFileToWrite`?

Reply:
Yeah, do you think I should make a function and just use it in the tests?

Reply:
ya since this is some sort of constant, i would recommend putting it somewhere that can be resused both in the tests and inside `generate`. Then you can also comment on what its used for for better clarity, cause right now its hard to tell what its doing

Reply:
If it's used in the tests though is it really testing that it was created correctly though? Because if alter it to incorrect code in the file it would change in the test too and pass incorrectly.